ern-container-gen-android/src/hull/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3-all.zip
+networkTimeout=10000
```

Comment:
It looks like RN 0.68.x comes with Gradle 7.3.3 (not 7.3) - Please ensure the version matches.

Reply:
I picked up this change from changelog https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0680. But i did check the branch for 0.68 and I see 7.3.3